corehq/apps/groups/dbaccessors.py

```diff
@@ -0,0 +1,45 @@
+import settings
+
+
+def group_by_domain(domain, include_docs=True, **kwargs):
+    from corehq.apps.groups.models import Group
+    return Group.view(
+        'groups/by_domain',
+        key=domain,
+        include_docs=include_docs,
+        **kwargs
+    )
```

Comment:
Totes agree.  One pattern I've been using in dbaccessors is to make a private master function for a particular couch view which parameterizes everything.  The public functions then all call that master function with the appropriate parameters.

This gives you the same constrained, testable public interface, while letting you reuse view code.  Also, when you *do* need to change that master function, all the usages of it are right there in the same dbaccessors file, and they all have tests.